### PR TITLE
Fix the issue #62 in the branch DEV

### PR DESCRIPTION
**Pull request by mniak**
Create method `Read(StreamReader)` in the class `FixedLengthFileMultiEngine` and exposes it in the `IFlatFileMultiEngine` interface

### CODE REVIEW

**Comment by mniak** [comment]
The build failed, but it's not caused by the changes I made.
AppVeyor says that `nuget restore` failed, but as you can see, I've not changed anything related to NuGet.

```
Error: 12/26/2017 1:37:07 PM: 
 +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [<<==>>] Exception: Exec: Error executing command  .$nuget restore $solution_path .
ERROR CODE: 1
Command exited with code 1
```

**Comment by mniak** [comment]
Tried to trigger build again but still not working.
I dont know what is happening but the code is fine

File: src/FlatFile.FixedLength/Implementation/FixedLengthFileMultiEngine.cs [hunk 1]
```diff
@@ -138,6 +138,27 @@ protected override bool TryParseLine<TEntity>(string line, int lineNumber, ref T
...
             var reader = new StreamReader(stream);
+            ReadInternal(reader);
+        }
+                
+        /// <summary>
+        /// Reads the specified stream reader.
+        /// </summary>
+        /// <param name="streamReader">The stream reader.</param>
+        /// <exception cref="ParseLineException">Impossible to parse line</exception>
+        public void Read(StreamReader reader)
```
Comment (niemyjski) [inline src/FlatFile.FixedLength/Implementation/FixedLengthFileMultiEngine.cs:149]: I'm not sure if this should be part of the contacts. We already have the ability to pass in a stream.

Comment (mthamil) [reply src/FlatFile.FixedLength/Implementation/FixedLengthFileMultiEngine.cs:149]: I actually would like the feature as well. I needed to use a custom StreamReader implementation for reading my files and ended up downcasting the engine to use that overload.

**Pull request closed without merging**